app/lib/database/utils.ts

```
@@ -1,4 +1,4 @@
-import { Q } from '@nozbe/watermelondb';
+import { Q } from './facade';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)